test/test_python_dispatch.py

```diff
@@ -69,6 +69,26 @@ def tearDown(self):
         if hasattr(torch.ops, self.test_ns):
             del torch.ops._test_python_registration
 
+    def test_fallback(self) -> None:
+        with _scoped_library("_", "IMPL") as my_lib:
+            orig_args = ((2, 2),)
+            orig_kwargs = {"device": torch.device("fpga"), "dtype": torch.float64}
+            def my_fallback(op, *args, **kwargs):
+                self.assertIs(op, torch.ops.aten.empty.memory_format)
+                self.assertEqual(args, orig_args)
+                # Why pin_memory was added?
+                self.assertEqual(kwargs, orig_kwargs | {"pin_memory": False})
+                # Any return is fine here
+                return torch.empty(*args)
+
+            # Use PrivateUse1 here as there are no key-specific kernels
+            # registered and so our fallback will always get called
+            my_lib.fallback(my_fallback, "FPGA")
```

Comment:
It might be better to testing something that isn't a factory function, because then the interaction with BackendSelect comes into play and I am not completely sure how that works (does BackendSelect know to redispatch to fpga)

Reply:
The problem is that I can't create an fpga Tensor actually. So I can't call any non-factory function on that device haha

> We can test this by adding the dispatch key to the local include set.

Should I use one the functorch key for example, add to the include set and use that?

Reply:
Added!